Docs/source/conf.py

```diff
@@ -34,7 +34,12 @@
 import sphinx_rtd_theme  # noqa
 from pybtex.style.formatting.unsrt import Style as UnsrtStyle
 
-sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../Regression/Checksum'))
```

Comment:
In this case, this would be cleaner if it used temporaries, like this
```
module_path = os.path.dirname(os.path.abspath(__file__))
checksum_path = os.path.join(module_path, "../../Regression/Checksum")
sys.path.insert(0, checksum_path)
```
But of course, ruff couldn't do this so this would have to be a separate PR.

Reply:
I agree, we can apply this manually here :+1:

Reply:
Applied :)